biojava-structure/src/main/java/org/biojava/nbio/structure/basepairs/MismatchedBasePairParameters.java

```diff
@@ -0,0 +1,186 @@
+/*
+ *                    BioJava development code
+ *
+ * This code may be freely distributed and modified under the
+ * terms of the GNU Lesser General Public Licence.  This should
+ * be distributed with the code.  If you do not have a copy,
+ * see:
+ *
+ *      http://www.gnu.org/copyleft/lesser.html
+ *
+ * Copyright for this code is held jointly by the individual
+ * authors.  These should be listed in @author doc comments.
+ *
+ * For more information on the BioJava project and its aims,
+ * or to join the biojava-l mailing list, visit the home page
+ * at:
+ *
+ *      http://www.biojava.org/
+ *
+ *
+ */
+package org.biojava.nbio.structure.basepairs;
+
+import org.biojava.nbio.structure.Atom;
+import org.biojava.nbio.structure.Chain;
+import org.biojava.nbio.structure.Group;
+import org.biojava.nbio.structure.Structure;
+import org.biojava.nbio.structure.contact.Pair;
+
+import javax.vecmath.Matrix4d;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This class allows for finding inter-strand base pairs that are not necessarily canonical Watson-Crick pairs.
+ * The implementation of findPair is different than that of the base class.  This class does not consider intra-strand
+ * base pairing and for that, the TertiaryBasePairParameters class should be used.
+ * @author Luke Czapla
+ * @since 5.0.0
+ *
+ */
+public class MismatchedBasePairParameters extends BasePairParameters implements Serializable {
+
+    // These are the criteria used to select proper base pairs.
+    protected static double MaxStagger = 2.0, MaxShear = 5.0, MaxStretch = 5.0,
```

Comment:
They have getters and setters actually, so they are not actually constants.  If someone really knows what they're doing, they can change these criteria.. I was just talking to Wilma Olson (a professor at Rutgers) about this in her office because I was down there for a teachers conference.  There are many different ways people choose the criteria for a base pair, she seemed to think this was good and showed me exactly what kinds of "base pairs" get taken out by the particular Stagger criteria I used.  It was stuff that shouldn't really belong as characterized as a base pair.